resources/test_site/mac_addrs/3c5ab41e8f0b/device_config.json

```diff
@@ -4,6 +4,15 @@
     "serial": "28491384"
   },
   "modules": {
+    "dot1x": {
+      "enabled": true,
+      "radius_server": {
+          "radius_socket_info": {
+              "remote_ip": "10.20.0.3",
```

Comment:
this might be unstable, but if it works it's fine.  There's no guarantee it will be at that address. Ideally DNS would work but I'm not sure if that's setup properly!